### PR TITLE
Pia 1078

### PR DESCRIPTION
**Pull request by zladzeyka**
Documentation:
  *  Updated section about example apps

### CODE REVIEW

File: Documentation/sections/Documentation.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ By integrating this SDK into your application you can allow your users to easily
...
 Further documentation with installation, integration or customization guides can be found in our [website](http://developer.gini.net/gini-pay-bank-sdk-ios/docs/).
 
-## Example
+## Example apps
 
-We are providing example app for integrating payment functionality for Swift. This app demonstrates how to integrate the Gini Pay Bank SDK. To run the example project, clone the repo and run `pod install` from the Example directory first.
-To inject your API credentials into the Example app, just add to the Example/Bank directory the `Credentials.plist` file with the following format:
+We are providing Bank and Business example apps for integrating payment functionality. Bank app demonstrates how to integrate the Gini Pay Bank SDK. Business app initiates the payment flow.
+To run the apps, clone the repo and run `pod install` from the Example directory first.
+To inject your API credentials into Business and Bank, just add to the Example/Bank and Example/Business directory the `Credentials.plist` file with the following format:
```
Comment (tastyminerals) [inline Documentation/sections/Documentation.md:24]: Less German :) `add the `Credentials.plist` file to the Example/Bank and Example/Business directory. The file should follow the format:`

File: Documentation/sections/Documentation.md [hunk 2]
```diff
@@ -17,10 +17,11 @@ By integrating this SDK into your application you can allow your users to easily
 
 Further documentation with installation, integration or customization guides can be found in our [website](http://developer.gini.net/gini-pay-bank-sdk-ios/docs/).
 
-## Example
+## Example apps
 
-We are providing example app for integrating payment functionality for Swift. This app demonstrates how to integrate the Gini Pay Bank SDK. To run the example project, clone the repo and run `pod install` from the Example directory first.
-To inject your API credentials into the Example app, just add to the Example/Bank directory the `Credentials.plist` file with the following format:
+We are providing Bank and Business example apps for integrating payment functionality. Bank app demonstrates how to integrate the Gini Pay Bank SDK. Business app initiates the payment flow.
+To run the apps, clone the repo and run `pod install` from the Example directory first.
```
Comment (tastyminerals) [inline Documentation/sections/Documentation.md:23]: It's nitpicking although `To` -> `In order to`.

File: Documentation/source/Integration.md [hunk 3]
```diff
@@ -68,16 +68,19 @@ bankSDK.receivePaymentRequest(paymentRequestId: appDelegate.paymentRequestId)
...
 ##  Resolving payment request
+
 ```swift
 
 bankSDK.resolvePaymentRequest(paymentRequesId: appDelegate.paymentRequestId,
                                  paymentInfo: paymentInfo)
 
 ```
-The method above returns the completion block with the struct `ResolvedPaymentRequest`, which includes requesterUri for redirecting back to the busithe payment requester app.
+The method above returns the completion block with the struct `ResolvedPaymentRequest`, which includes requesterUri for redirecting back to the payment requester's app.
```
Comment (tastyminerals) [inline Documentation/source/Integration.md:80]: Does "requesterUri" miss the ticks?

File: README.md [hunk 4]
```diff
@@ -21,8 +21,10 @@ Further documentation with installation, integration or customization guides can
 
 ## Example
 
-We are providing example app for Swift. This app demonstrates how to integrate the Gini Pay Bank SDK with the Screen API and Component API. To run the example project, clone the repo and run `pod install` from the Example directory first.
-To inject your API credentials into the Example app, just add to the Example directory the `Credentials.plist` file with the following format:
+We are providing Bank and Business example apps for integrating payment functionality. Bank app demonstrates how to integrate the Gini Pay Bank SDK. Business app initiates the payment flow.
```
Comment (tastyminerals) [inline README.md:24]: `The Bank app`

File: Documentation/sections/Documentation.md (hunk 1 above)
Comment (a-szotyori) [reply Documentation/sections/Documentation.md:24]: Could you add these Credentials.plist files with placeholder content? Then clients would need to only fill it out, like [here](https://github.com/gini/gini-pay-business-sdk-ios/blob/master/Example/Example%20Swift/Credentials.plist). Then we also don't need the format information here.
```
To inject your API credentials into the Business and Bank example apps you need to add your credentials to `Example/Business/Credentials.plist` and `Example/Bank/Credentials.pllist`, respectively.
```

Comment (zladzeyka) [reply Documentation/sections/Documentation.md:24]: files already there, I'll update the text

**Review by a-szotyori — APPROVED** [review]

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**